extensions/amp-analytics/0.1/test/test-amp-analytics.js

```diff
@@ -208,9 +208,11 @@ describes.realWin('amp-analytics', {
       actualResults[vendor] = {};
       describe('analytics vendor: ' + vendor, function() {
         beforeEach(() => {
-          if (!config.triggers) {
-            expectAsyncConsoleError(noTriggersError);
-          }
+          // Remove all the triggers to prevent unwanted requests, for instance
+          // one from a "visible" trigger. Those unwanted requests are a source
+          // of test flakiness. Especially they will alternate value of var
+          // $requestCount.
+          config.triggers = {};
```

Comment:
Makes me think if we should also throw an user error if triggers is invalid or empty.

Reply:
We already print a user error if "trigger" is empty. amp-analytics.js L234

Reply:
That line checks for `!this.config_['trigger']` but doesn't check for `triggers: {}` or `triggers: {'invalidName': {}}`